app/views/org_admin/templates/index.html.erb

```diff
@@ -25,15 +25,18 @@
     <ul class="nav nav-tabs" role="tablist">
       <% if current_user.can_super_admin? %>
         <li role="all-templates" class="active">
-          <a href="#all-templates" role="tab" aria-controls="all-templates" data-toggle="tab"><%= _('All Templates') %></a>
+          <a data-target="#all-templates" role="tab" aria-controls="all-templates" data-toggle="tab"><%= _('All Templates') %></a>
         </li>
       <% end %>
       <li role="organisation-templates" class="<%= !current_user.can_super_admin? ? 'active' : '' %>">
-        <a href="#organisation-templates" role="tab" aria-controls="organisation-templates" data-toggle="tab"><%= current_user.can_super_admin? ? _('%{org_name} Templates') % { org_name: current_user.org.name } : _('Own Templates') %></a>
-      </li>
-      <li role="funder-templates">
-        <a href="#funder-templates" role="tab" aria-controls="funder-templates" data-toggle="tab"><%= _('Customizable Templates') %></a>
+        <a data-target="#organisation-templates" role="tab" aria-controls="organisation-templates" data-toggle="tab"><%= current_user.can_super_admin? ? _('%{org_name} Templates') % { org_name: current_user.org.name } : _('Own Templates') %></a>
       </li>
+      <!-- If the Org is not just a funder then show the customizations table -->
+      <% if !current_org.funder_only? %>
```

Comment:
funder_only checks if the org_type is strictly funder however the org could be funder and something else too. I would make sure that org_type is not funder, i.e. the bit flag does not include funder type at all. What do you think?

Reply:
we actually only want to hide the tab if the Org is only a funder (so its ok to show for funder/organisations, etc.)

Reply:
we only want to hide the customizable templates table if the the org is just a funder, so if they were a funder+organization we want them to see it.